<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BBProjectDialog</class>
 <widget class="QWidget" name="BBProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#topWidgetBackground {
    background-color: rgb(14, 19, 25);
}

#iconBox {
    border: 2px solid #0ebf9c;
    border-radius: 6px;
}

#icon {
    border-image: url(../../resources/icons/face.png);
    border-radius: 3px;
    margin: 3px;
}

#name {
    color: #d6dfeb;
    font: 9pt &quot;Arial&quot;;
}

#introduction {
    color: #d6dfeb;
    font: 10pt &quot;Arial&quot;;
}

#buttonFolder, #buttonNew, #buttonOpen, #buttonClose, #buttonMinimize, #buttonMaximize {
    border: none;
    border-radius: 6px;
    color: rgb(56, 70, 87);
    background-color: #d6dfeb;
}

#buttonFolder:pressed, #buttonNew:pressed, #buttonOpen:pressed,
#buttonFolder:hover, #buttonNew:hover, #buttonOpen:hover {
    background-color: #8193bc;
}

#buttonFolder {
    image: url(../../resources/icons/folder.png);
}

#buttonNew {
    image: url(../../resources/icons/new.png);
}

#buttonClose {
   background-color: rgb(252, 98, 93);
}

#buttonClose:pressed {
   background-color: rgb(239, 78, 73);
}

#pageFolder {
    background-color: rgb(14, 19, 25);
}

#pageNew {
    background-color: rgb(14, 19, 25);
}

#buttonDisk, #buttonCloud {
    color: #d6dfeb;
    text-align: right;
    font: 9pt &quot;Arial&quot;;
    background: rgb(36, 43, 58);
    border: none;
    border-bottom-left-radius: 6px;
    border-top-left-radius: 6px;
    padding: 8px;
}

#buttonDisk:hover, #buttonCloud:hover, #buttonDisk:checked, #buttonCloud:checked {
    color: #d6dfeb;
    text-align: right;
    font: 9pt &quot;Arial&quot;;
    background: #0ebf9c;
    border: none;
    border-radius: 0px;
    border-bottom-left-radius: 6px;
    border-top-left-radius: 6px;
    padding: 8px;
}

#buttonOpen {
    text-align: right;
    font: 9pt &quot;Arial&quot;;
    border-radius: 6px;
    padding: 8px;
    image: url(../../resources/icons/open.png);
}

QScrollBar:vertical {
    width: 8px;
    background: transparent;
}

QScrollBar:horizontal {
    height: 8px;
    background: transparent;
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #0ebf9c;
    border: none;
    border-radius: 4px;
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:horizontal:hover {
    background: #0ebf9c;
    border: none;
    border-radius: 4px;
}

QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical,
QScrollBar::sub-line:horizontal, QScrollBar::add-line:horizontal {
    width: 0;
    height: 0;
}

QListWidget, #pageNewBackGround {
    color: #d6dfeb;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #191f28, stop: 1 rgb(36, 43, 58));
    border: none;
    font: 9pt &quot;Arial&quot;;
}

QListWidget::item {
    padding: 3px;
    background: transparent;
}

QListWidget::item:selected:!active {
    background: #8193bc;
    color: #d6dfeb;
}

QListWidget::item:selected:active {
    background: #8193bc;
    color: #d6dfeb;
}

QListWidget::item:hover {
    background: #508193bc;
}

#labelProjectName, #labelLocation {
    font: 75 9pt &quot;Arial&quot;;
    color: #d6dfeb;
}

#lineEditProjectName, #lineEditLocation {
    color: #191f28;
    background: #d6dfeb;
    border: none;
    padding: 6px;
    font: 9pt &quot;Arial&quot;;
    selection-background-color: #8c0ebf9c;
    selection-color: #d6dfeb;
}

#buttonOpenLocation {
    color: #0ebf9c;
    border: none;
    background: none;
    font: 75 18pt &quot;Arial&quot;;
}

#editLocationWidget {
    border: none;
    background: #d6dfeb;
}

#buttonCancel {
    border: none;
    border-radius: 4px;
    padding: 6px;
    padding-left: 18px;
    padding-right: 18px;
    color: #d6dfeb;
    background: #8193bc;
    font: 9pt &quot;Arial&quot;;
}

#buttonCancel:hover {
    background: #8c8193bc;
}

#buttonCreate {
    border: none;
    border-radius: 4px;
    padding: 6px;
    padding-left: 18px;
    padding-right: 18px;
    color: #d6dfeb;
    background: #0ebf9c;
    font: 9pt &quot;Arial&quot;;
}

#buttonCreate:hover {
    background: #8c0ebf9c;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidgetBackground" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="topWidgetBackgroundHorizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QWidget" name="buttonGroup" native="true">
        <layout class="QVBoxLayout" name="buttonGroupVerticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="horizontalWidget" native="true">
           <layout class="QHBoxLayout" name="buttonGroupWindow">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="buttonClose">
              <property name="minimumSize">
               <size>
                <width>14</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>14</width>
                <height>14</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="introduction">
           <property name="text">
            <string>Project Management</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QWidget" name="toolbar3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignBottom">
          <widget class="QPushButton" name="buttonFolder">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignBottom">
          <widget class="QPushButton" name="buttonNew">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QWidget" name="user" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="name">
           <property name="text">
            <string>BigBear</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="iconBox" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="icon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedFolderAndNew">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageFolder">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QWidget" name="buttonGroupLeft" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="buttonDisk">
            <property name="text">
             <string>On Disk</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignTop">
           <widget class="QPushButton" name="buttonCloud">
            <property name="text">
             <string>In the Cloud</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignBottom">
           <widget class="QPushButton" name="buttonOpen">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedDiskAndCloud">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>720</width>
           <height>480</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageDisk">
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QListWidget" name="listDiskProject">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageCloud">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>particle</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageNew">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QWidget" name="pageNewBackGround">
         <layout class="QVBoxLayout" name="vLayoutPageNewBackGround">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignCenter">
           <widget class="QWidget" name="pageNewContent">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="vLayoutPageNewContent">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="labelProjectName">
               <property name="text">
                <string>Project Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditProjectName">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignLeft">
              <widget class="QLabel" name="labelLocation">
               <property name="text">
                <string>Location</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="editLocationWidget">
               <layout class="QHBoxLayout" name="hLayoutEditLocationWidget">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLineEdit" name="lineEditLocation">
                  <property name="focusPolicy">
                   <enum>Qt::ClickFocus</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonOpenLocation">
                  <property name="text">
                   <string>···</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QWidget" name="pageNewButtonWidget">
               <layout class="QHBoxLayout" name="hLayoutPageNewButtonWidget">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>10</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="buttonCancel">
                  <property name="text">
                   <string>Cancel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonCreate">
                  <property name="text">
                   <string>Create Project</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
